### PR TITLE
ci: update Snyk generated PR titles to have a Conventional Commit prefix

### PR DESCRIPTION
**Pull request by jmartin4563**
## Links
Closes NR-96232

## Details
Updates any PR that is generated by [Snyk Bot](https://github.com/snyk-bot) to add a `security:` prefix to the title. Snyk does not give us a way to configure the commit message or customize anything about the PR like Dependabot does, so since we're going to eventually update the PR process to use squash/merge with the PR title as the commit message, this (in a roundabout way) gets our Snyk PRs to be "conventional", and ensure that they end up in the appropriate release notes section.

### CODE REVIEW

File: bin/update-snyk-pr.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * Copyright 2023 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const Github = require('./github')
+const SECURITY_PREFIX = 'security:'
+
+module.exports = async function updateSnykPR() {
```
Comment (bizob2828) [inline bin/update-snyk-pr.js:11]: I would do something like this so you don't have to do that weird `node -e 'require` in the workflow.

```js
-module.exports = async function updateSnykPR() {
+async function updateSnykPR() {
   const org = process.env.RELEASE_ORG || 'newrelic'
   const repo = process.env.RELEASE_REPO || 'node-newrelic'
   const prId = process.env.SNYK_PR_ID
@@ -29,3 +29,10 @@ module.exports = async function updateSnykPR() {
 
   await github.updatePullRequest({ id: prId, title: newTitle })
 }
+
+if (require.main === module) {
+  updateSnykPR()
+} else {
+  module.exports = updateSnykPR
+}

```

**Review by bizob2828 — CHANGES_REQUESTED** [review]

**Review by bizob2828 — APPROVED** [review]
Unrelated but I noticed the `npm run unit:scripts` runs in both lint and ci jobs, is this intentional?

**Comment by jmartin4563** [comment]
> Unrelated but I noticed the `npm run unit:scripts` runs in both lint and ci jobs, is this intentional?

It is not intentional, I originally was just going to add it to the lint staged, but didn't like how that looked in GHA, so added a new step, I'll remove the lint remnant

**Review by bizob2828 — APPROVED** [review]

**Comment by bizob2828** [comment]
looks good just wait to merge until we release

**Pull request merged**